Hymns/Browse/TagListView.swift

```diff
@@ -11,11 +11,18 @@ struct TagListView: View {
 
     var body: some View {
         VStack {
+            if viewModel.tags.isEmpty {
+                VStack {
+                    Image("empty tag illustration")
+                    Text("Create tags by tapping on the tag icon on any hymn").padding().multilineTextAlignment(.center)
+                }.frame(minWidth: 0, maxWidth: .infinity, minHeight: 0, maxHeight: .infinity, alignment: .center).background(Color(.systemBackground))
```

Comment:
Do you need the `.systemBackground`? Is it not that already by default?

Reply:
No, it's transparent. Remember we had this issue before with Chords, Guitar, and Piano. All which where under the Indicator Tab View.

Reply:
ah, ok. that's fine then